scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -159,6 +161,21 @@ namespace <%= namespace %> {
         }
 
         /// <summary>
+        /// Launches an external browser with the Cart's web checkout link loaded. This can be used to perform
+        /// a cart checkout when no native web view is available, such as on desktop, or from the Unity editor.
+        /// </summary>
+        /// <param name="success">called when the game has regained focus and the checkout was successful.</param>
+        /// <param name="cancelled">called when the game has regained focus but the checkout has not completed.</param>
+        /// <param name="failure">called when the game has regained focused and an error has occured during checkout.</param>
+        public void CheckoutWithExternalBrowser(CheckoutSuccessCallback success, CheckoutCancelCallback cancelled, CheckoutFailureCallback failure) {
```

Comment:
I'm kind of leaning towards Option 3 since there isn't much a difference between an external browser and the browsers we show on mobile. Technically we don't show a 'web view' on either platform - we launch into a browser. I wonder if a developer would ever want to differentiate between the external browser and the browsers we launch on mobile. I can see it being an implementation detail on our side. To them I can see if being the "non-customizable web checkout". Maybe we rename this method to be `performWebCheckout` to wrap both approaches.

The only caveat is we would need to be able to reliably call the success/failure delegates in the same way across both approaches (which I think we can do?)